### ISSUE TITLE
Results of interest for 2024-02-15 meeting

### ISSUE THREAD

**Issue by dchiu911** [TalhoukLab/OV_Histotypes#47]
- Table 3.17: Pre-QC Training Set Histotype Distribution by CodeSet
- Table 3.20: Number of failed sampled by CodeSet
- Figure 3.2: % Genes Detected vs. Signal to Noise Ratio (Zoomed)
- Table in Section 4.6: has the F1-score IV summaries of all classifiers used
- Figure 4.27: Gene Optimization for Sequential Classifier
- Figure 4.28: Gene Optimization for Two-Step Classifier
- Figure 4.29: Top 4 Model Evaluation Metrics
- Figure 4.31: Top 4 Model Per-Class F1-Scores
- Table 4.26: Class-specific F1-scores on Confirmation Set Models
- Table 4.27: Class-specific F1-scores on Validation Set Model
